cumulusci/tasks/metadata_etl/field_sets.py

```diff
@@ -0,0 +1,74 @@
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.metadata_etl import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+from cumulusci.core.utils import process_bool_arg, process_list_arg
+
+from logging import getLogger
+
+logger = getLogger(__name__)
```

Comment:
Tasks already get their own loggers in `self.logger` - it's set up in the `BaseTask` class.